config.py

```diff
@@ -890,18 +900,19 @@ def get_checks_places(osname, agentConfig):
         log.error(e.args[0])
         sys.exit(3)
 
-    places = [lambda name: os.path.join(agentConfig['additional_checksd'], '%s.py' % name)]
+    places = [lambda name: (os.path.join(agentConfig['additional_checksd'], '%s.py' % name), None)]
 
     try:
         if Platform.is_windows():
-            places.append(get_windows_sdk_check)
+            places.append(lambda name: (get_windows_sdk_check(name), None))
```

Comment:
You should add this to the get_windows_sdk_check function. It should be relatively easy to return the tuple there as well.